webui/react/src/types.ts

```diff
@@ -659,3 +646,15 @@ export interface PriorityScheduler {
   defaultPriority: number;
   preemption: boolean;
 }
+
+/* Jobs */
+
+export type Job = Api.V1Job;
+export type JobType = Api.Determinedjobv1Type;
+export type JobState = Api.Determinedjobv1State;
+export type JobSummary = Api.V1JobSummary;
+export type RPStats = Api.V1RPQueueStat;
+
+export type JobAction = 'Cancel'| 'Kill'| 'Manage Job'| 'Move to Top'
```

Comment:
**nit:** equal spacing between the single pipes?
```
export type JobAction = 'Cancel' | 'Kill' | 'Manage Job' | 'Move to Top'
```
not sure if [this lint rule](https://eslint.org/docs/rules/space-infix-ops) will fix it

Reply:
that seems to do what we want but also comes with spacing for other operators and results in about 50 lines of change. I'll leave that off for another pr.

Reply:
https://github.com/determined-ai/determined/pull/3200